Assets/Scripts/UI/PauseScrn/Navigation_Notebook.cs

```diff
@@ -25,11 +25,6 @@ void Start()
 		note2.SetActive (false);
 		note3.SetActive (false);
 		note4.SetActive (false);
-
-		if (note0.activeSelf == false && note1.activeSelf == false && note2.activeSelf == false && note3.activeSelf == false && note4.activeSelf == false) {
```

Comment:
Dont we need this? When we dont have any notebook the back should always be selected right?